app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -162,7 +162,16 @@ abstract class TwoPaneFragment : Fragment() {
 
     override fun onConfigurationChanged(newConfig: Configuration) {
         super.onConfigurationChanged(newConfig)
+
         ensureThreadIsOpen()
+
+        val isLeftShown = areBothShown() || isOnlyLeftShown() // TODO: Only works on Phone, and not on Tablet.
```

Comment:
Currently the naming feels kind of confusing, is there no better way of naming things?

Reply:
I personnaly don't find it confusing